marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/job/startJob.sjs

```diff
@@ -14,24 +14,39 @@
  limitations under the License.
  */
 'use strict';
-
 xdmp.securityAssert("http://marklogic.com/data-hub/privileges/run-step", "execute");
 
 const consts = require('/data-hub/5/impl/consts.sjs');
+const config = require("/com.marklogic.hub/config.sjs");
 const hubUtils = require("/data-hub/5/impl/hub-utils.sjs");
 const Job = require("/data-hub/5/flow/job.sjs");
 const jobs = require("/data-hub/5/impl/jobs.sjs");
+const dhProv = require('/data-hub/5/provenance/dh-provenance.xqy');
 
 var jobId;
 var flowName;
 
+function recordJobProvenance(jobId, timeStarted) {
+  const options = {
+    "startTime": timeStarted,
```

Comment:
I highly recommend using "startDateTime" to make it clear that this is a dateTime value. If it's a date, do "startDate". If it's a time - e.g. "HH:mm:ss" - call it "startTime". Using explicit naming like that removes any uncertainty over whether the expected value is a date, time, or dateTime.